control-plane/csi-driver/src/bin/node/main_.rs

```diff
@@ -440,17 +474,13 @@ impl CsiServer {
                 debug!("Successfully changed file permissions for CSI socket");
             }
 
-            async_stream::stream! {
-                loop {
-                    let item = uds.accept().map_ok(|(st, _)| UnixStream(st)).await;
-                    yield item;
-                }
-            }
+            UnixListenerStream::new(uds)
         };
 
         let node = Node::new(node_name.into(), node_selector, probe_filesystems());
         Ok(async move {
             Server::builder()
+                .timeout(std::time::Duration::from_secs(30))
```

Comment:
Should this be configurable?

Reply:
As discussed, we can even not have this at all if not required

Reply:
Yeah we can, although kubelet cancels it in less than 2mins either way as we have seen, so that should be good enough?

Reply:
Removed it.